DEVELOPMENT.md

```diff
@@ -0,0 +1,67 @@
+## Development
```

Comment:
Developers should read the development section of the website, which covers thing like development philosophy and contribution process https://prestosql.io/development/